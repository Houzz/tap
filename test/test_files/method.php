<?php

class A {
    public function func() {
    }
}

function testMethod() {
    $a = new A();
    $a->func();
    $a->func(1);    // error
    $a->notExisting();  // error
    $b = new B();   // error
}


