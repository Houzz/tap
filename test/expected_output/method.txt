11 E_FUNCTION_ARGUMENT
12 E_METHOD_NOT_FOUND
13 E_CLASS_NOT_FOUND
